data/exceptions.json

```diff
@@ -145,9 +145,18 @@
   {"period": "Lunch", "time": "11:45-12:30"}
 ]},
 "NoSchool":{
-  "MS": [],
-
-  "US": []
+  "allDayEvent": {
+    "name": "noschool",
```

Comment:
|name| should be the displayed name.
|imagePrefix| should be the prefix for the image file name.
I would use numImages instead of |imageNum|; |imageNum| sounds like the number of a specific image.